makefile

```diff
@@ -2,8 +2,20 @@
 
 all: requirements static plugins
 
-test: all
-	python setup.py test
+es:
+	cd docker_es; docker build -t es_dxr .; docker rm -f es_dxr || true; docker run --name es_dxr -p 80:8000 es_dxr
```

Comment:
I assume these are all on one line because docker twiddles some env vars? If we do have to keep them on one line, we should at least separate the commands with `&&` so they don't plow onward if something breaks.